source/features/show-names.tsx

```diff
@@ -24,10 +24,15 @@ async function dropExtraCopy(link: HTMLAnchorElement): Promise<void> {
 function appendName(element: HTMLAnchorElement, fullName: string): void {
 	// If it's a regular comment author, add it outside <strong> otherwise it's something like "User added some commits"
 	const {parentElement} = element;
-	const insertionPoint = parentElement!.tagName === 'STRONG' ? parentElement! : element;
+	const isStrong = parentElement?.tagName === 'STRONG';
 
+	if (isStrong) {
+		parentElement?.classList.add('inline');
+	}
+
+	const insertionPoint = isStrong ? parentElement! : element;
 	insertionPoint.after(
-		' ',
+		<>&nbsp;</>,
```

Comment:
In new issue view the first space we need to use `&nbsp;`, because whitespace is not work here. And the second space in new issue view we don't need to care about it. Its parent have flex attribute. If we do same thing in second space it will have more space than one whitespace.

Reply:
Dont't need this.